toolkit/tools/internal/resources/assets/grub2/grub.cfg

```diff
@@ -0,0 +1,26 @@
+set timeout=0
+set bootprefix={{.BootPrefix}}
+search -n -u {{.BootUUID}} -s
+
+load_env -f $bootprefix/mariner.cfg
```

Comment:
Worth noting that these .cfg files only are present if we are installing 2.0 (.cm2) packages (specifically the kernel, systemd) 

Can we add a sanity check when includeLegacyGrubCfg=true that we are modifying a 2.0 image?

Reply:
Considering that only the Image Customizer is passing in `true`, I think it makes sense to leave the image version detection to the Image Customizer code.

Reply:
And in fact, I'll probably do this as a detection of if the image is using grub-mkconfig.